pkg/restapi/issuer/operation/operations.go

```diff
@@ -152,11 +196,62 @@ func (c *Operation) Callback(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := t.Execute(w, vc{Data: string(cred)}); err != nil {
+	// retrieve credential returns the strings with back slashes, to render the response as string without slashes
+	cr, err := strconv.Unquote(string(cred))
```

Comment:
@DRK3 we need to fix this issue in edv

Reply:
@fqutishat OK. I'm going to look at it right now